packages/contract-schema/test/schema.js

```diff
@@ -3,20 +3,43 @@ var assert = require("assert");
 
 var MetaCoin = require("./MetaCoin.json");
 
-describe("Schema", function() {
-  it("validates correct input", function() {
+describe("Schema", function () {
+  it("validates correct input", function () {
     Schema.validate(MetaCoin);
   });
 
-  it("throws exception on invalid input", function() {
+  it("throws exception on invalid input", function () {
     var invalid = {
       abi: -1
     };
 
     try {
       Schema.validate(invalid);
     } catch (err) {
-      var abiErrors = err.errors.filter(function(error) {
+      var abiErrors = err.errors.filter(function (error) {
+        return error.dataPath === ".abi";
+      });
+      assert(abiErrors);
```

Comment:
Oh, Amal is absolutely right here, I totally missed this!  Note that also you'll need to check that `err.errors` exists before using it.

Reply:
@cds-amal Thank you for heads-up! Fixed.

@haltman-at `err.errors` being undefined would not imply that `Ajv.validate()` does not work as expected, and should be treated as a failing test? 🤔

Reply:
@tgbv Strictly speaking that is true, but IMO test failures should come either from the code being tested, or from a deliberate assert.  An exception originating in the test itself, that is not a deliberate assertion, suggests that something is wrong with the test, rather than with the code being tested.